clientd3d/d3dparticle.c

```diff
@@ -97,6 +115,16 @@ void D3DParticleSystemUpdate(particle_system *pParticleSystem, d3d_render_pool_n
 			}
 			else
 			{
+				PDIB pdibCeiling = NULL;
```

Comment:
This NULL doesn't do anything; combine with the next line.

(Also prefer nullptr to NULL, as NULL is an ugly macro.)